hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/parse/SqlCreateMappingTest.java

```diff
@@ -46,14 +46,15 @@ public void test_unparse() {
         );
 
         String sql = SqlCreateMapping.unparse(mapping);
-        assertThat(sql).isEqualTo("CREATE MAPPING \"name\" EXTERNAL NAME \"external-name\" (" + LE +
+        assertThat(sql).isEqualTo("CREATE OR REPLACE EXTERNAL MAPPING \"hazelcast\".\"public\".\"name\" " +
```

Comment:
`CREATE OR REPLACE EXTERNAL MAPPING`? there is no `EXTERNAL` there in the syntax

Reply:
at least in https://docs.hazelcast.com/hazelcast/5.3-snapshot/sql/create-mapping

Reply:
![Uploading image.png…]()